### PR TITLE
Shorten Settings editor aria labels

### LINKED ISSUE #153588: Modified Elsewhere aria label should be more concise
Testing #153351

Currently the screen reader reads out  "The default value of the setting has also been overridden for the following languages: LANG1, LANG2".

We should always strive towards concisnes for aria labels. So let's go with something like:
"LANG1, LANG2... have overridden this setting"

Similar for other cases. Just try to keep the label as concise as possible :)

### PR DESCRIPTION
**Pull request by rzhao271**
This PR fixes #153588

### CODE REVIEW

File: src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts [hunk 1]
```diff
@@ -290,21 +290,21 @@ export function getIndicatorsLabelAriaLabel(element: SettingsTreeSettingElement,
...
+		ariaLabelSections.push(localize('defaultOverriddenDetailsAriaLabel', "{0} overrides the default value", sourceToDisplay));
 	}
 
 	// Add text about default values being overridden in other languages
-	const otherLanguageOverridesStart = localize('defaultOverriddenListPreface', "The default value of the setting has also been overridden for the following languages:");
 	const otherLanguageOverridesList = element.overriddenDefaultsLanguageList
 		.map(language => languageService.getLanguageName(language)).join(', ');
 	if (element.overriddenDefaultsLanguageList.length) {
-		ariaLabelSections.push(`${otherLanguageOverridesStart} ${otherLanguageOverridesList}`);
+		const otherLanguageOverridesText = localize('defaultOverriddenLanguagesList', "Language-specific default values exist for {0}", otherLanguageOverridesList);
```
Comment (rzhao271) [inline src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts:306]: Do we have a pluralization module? Or, does `{0} contribute(s) language-specific default values` with the `(s)` work?

Comment (alexr00) [reply src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts:306]: Not that I know of. The way I've done it in the past is to have two strings:
```
{0} contribute language-specific default values
{0} contributes language-specific default values
```

**Review by alexr00 — APPROVED** [review]

File: src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts (hunk 1 above)
Comment (isidorn) [reply src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts:306]: Yup. No helper method that I am aware of.

**Review by isidorn — APPROVED** [review]

File: src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts (hunk 1 above)
Comment (rzhao271) [reply src/vs/workbench/contrib/preferences/browser/settingsEditorSettingIndicators.ts:306]: Will switch to using two strings during debt week.

**Pull request merged**